index.js

```diff
@@ -1,88 +1,119 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
-
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+console.log("%cQuestion 1", "color: red") 
+const counter = 2
```

Comment:
We want to be able to reassign the `counter` variable.
At the moment, because we are using `const`, we can't reassign the variable. :(